bindings/trycp_runner/src/common.rs

```diff
@@ -466,3 +467,27 @@ where
             .map_err(|e| anyhow::anyhow!("Decoding failure: {:?}", e))
     })
 }
+
+/// Call [`run`] for a scenario and check that it completed with the minimum required agents.
+///
+/// The value of `min_required_agents` can be overridden with the environment variable
+/// `MIN_REQUIRED_AGENTS` when running a scenario.
+pub fn run_with_required_agents<RV: UserValuesConstraint, V: UserValuesConstraint>(
+    definition: ScenarioDefinitionBuilder<RV, V>,
+    min_required_agents: usize,
+) -> anyhow::Result<()> {
+    let agents_at_completion = run(definition)?;
+
+    let min_required_agents = std::env::var("MIN_REQUIRED_AGENTS")
+        .ok()
+        .and_then(|v| v.parse().ok())
+        .unwrap_or(min_required_agents);
```

Comment:
Do we care about any of these errors? Right now if `MIN_REQUIRED_AGENTS` is not set or is not a valid number then it just silently uses the passed in value. Is this desired behaviour?

Reply:
I'd probably just log a debug message in the unwrap to say that the default value is being used, then it's easy enough to figure out what's going on

Reply:
I added a `debug` if it's not set at all because this is probably the expected path but then a `warn` if it's an invalid value.